test/blackbox-tests/gen_tests.ml

```diff
@@ -60,7 +60,7 @@ module Test = struct
         ; atom (sprintf "test-cases/%s" t.name)
         ; List
             [ atom "progn"
-            ; Sexp.strings (["run"; "${exe:cram.exe}"]
+            ; Sexp.strings (["run"; "%{exe:cram.exe}"]
```

Comment:
This doesn't seem right. `Sexp.atom_or_quoted_string` should always produce something that parses back as an atom or quoted string, however here we use it to produce something that will parse as a template.

Reply:
Yeah, I guess should just use the template API then.